internal/cmd/manager/instance/restore/cmd.go

```diff
@@ -32,6 +32,8 @@ func NewCmd() *cobra.Command {
 	var clusterName string
 	var namespace string
 	var pgData string
+	// TODO: add support
+	var pgWal string
```

Comment:
do we need to do anything special for the restore cmd?

Reply:
IIRC barman-cloud-restore will not restore the WAL files but create an empty folder. Anyway we should certainly override this folder with a symlink pointing to the correct location.